uli_init/simulate.py

```diff
@@ -704,40 +700,41 @@ def build_molecule(molecule, length, para_weight):
     sequence : list
         List of the configuration sequence of the finished compound
     """
-    f = open("{}/{}.json".format(COMPOUND_DIR, molecule))
+    f = open(f"{COMPOUND_DIR}/{molecule}.json")
```

Comment:
~one question I have here -- are the files always .json? A user would never want to load one of the mol2 files? e.g., `uli_init/compounds/peek-para.mol2`?~
nevermind! I see now that the json is required because it also has the bond indices, etc.